Sources/Fullscreen/TransactionView/TransactionView.swift

```diff
@@ -178,6 +183,27 @@ private extension TransactionView {
         connector.connect(from: stepDot, to: previousStepDot)
     }
 
+    func setupVerticalStackViewContraints() {
+        guard
+            let firstStepDot = stepDots.first,
+            let lastTransactionStepView = verticalStackView.arrangedSubviews.last
+        else { return }
+
+        verticalStackViewLeadingAnchor = verticalStackView.leadingAnchor.constraint(equalTo: firstStepDot.trailingAnchor, constant: .spacingS)
+        verticalStackViewBottomAnchor = verticalStackView.bottomAnchor.constraint(equalTo: lastTransactionStepView.bottomAnchor)
+
+        NSLayoutConstraint.activate([
+            verticalStackViewLeadingAnchor!,
```

Comment:
is `!` necessary?

Reply:
The value is set two lines above, I don't see a need to guard it?